bazaarvoice/emodb
# Make Consumer Group Names Configurable

## Github Issue #

`None`

## What Are We Doing Here?

We are making consumer group names configurable so that they can comply with cluster convention.

We are also doing some package relocation as a result. My apologies 🙏 .

## How to Test and Verify

Run the megabus and confirm that the consumer group names that are configured match what is in zookeeper/kafka.

## Risk

### Level 

`Low`

### Required Testing

`Manual`

### Risk Summary

The only risk here is that we are giving the configurator the ability cause consumer group intersection by giving two different megabuses the same consumer group name.

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
